worker-markup-core/src/main/java/com/github/cafdataprocessing/worker/markup/core/MarkupHeadersAndBody.java

```diff
@@ -180,6 +180,24 @@ else if (line.contains(":*")) {
             }
         }
         LOG.info("Markup of Headers and Body complete");
+
+        // Fix for SCMOD-4128 to untag and detach erroneous emails which have blank headers.
```

Comment:
Don't mention the Jira number in the comments.  Just describe the behavior.  Describe for example why you are starting at index 1 rather than index 0.  A casual reader may think you've made a mistake with something unusual like that.